agentendpoint/agentendpoint.go

```diff
@@ -101,7 +102,20 @@ func (c *Client) RegisterAgent(ctx context.Context) error {
 		return err
 	}
 
-	req := &agentendpointpb.RegisterAgentRequest{AgentVersion: agentconfig.Version(), SupportedCapabilities: agentconfig.Capabilities()}
+	oi, err := osinfo.Get()
```

Comment:
Should we populate this only if inventory is turned on?

Reply:
Fixed